### PR TITLE
Add support to URL with empty scheme in http:Client

### PR DESCRIPTION
**Pull request by TharmiganK**
## Purpose

This adds default scheme `http://` when the scheme is omitted in the URL

> Fixes [`Append http:// when omitted in the http:client #1986`](https://github.com/ballerina-platform/ballerina-standard-library/issues/1986)

## Examples

Refer the issue

## Checklist
- [x] Linked to an issue
- [x] Updated the changelog
- [x] Added tests

### CODE REVIEW

File: ballerina-tests/tests/http_client_actions_test.bal [hunk 1]
```diff
@@ -617,9 +617,9 @@ function testClientPathWithWhitespaces() {
 
 @test:Config {}
 function testClientInitWithMalformedURL() {
-    http:Client|error httpEndpoint = new ("invalid URL here");
+    http:Client|error httpEndpoint = new ("httpeds://bar.com/foo");
```
Comment (chamil321) [inline ballerina-tests/tests/http_client_actions_test.bal:620]: Let's add a test case for empty url also if we have not any

File: native/src/main/java/io/ballerina/stdlib/http/api/client/endpoint/CreateSimpleHttpClient.java [hunk 2]
```diff
@@ -51,7 +51,13 @@ public static Object createSimpleHttpClient(BObject httpClient, BMap globalPoolC
...
             HttpConnectionManager connectionManager = HttpConnectionManager.getInstance();
             String scheme;
-            String urlString = clientUrl.getValue().replaceAll(HttpConstants.REGEX, HttpConstants.SINGLE_SLASH);
+            String urlString = clientUrl.getValue();
+            String[] urlStrings = urlString.split(HttpConstants.SCHEME_SEPARATOR, 2);
+            if (urlStrings.length == 1) {
+                urlStrings = new String[]{HttpConstants.HTTP_SCHEME, urlStrings[0]};
+            }
+            urlStrings[1] = urlStrings[1].replaceAll(HttpConstants.DOUBLE_SLASH, HttpConstants.SINGLE_SLASH);
+            urlString = urlStrings[0] + HttpConstants.SCHEME_SEPARATOR + urlStrings[1];
```
[the remark this answers is not in the document]
Comment (TharmiganK) [reply native/src/main/java/io/ballerina/stdlib/http/api/client/endpoint/CreateSimpleHttpClient.java:60]: Yes I missed that
I will correct it

[the remark this answers is not in the document]
Comment (TharmiganK) [reply native/src/main/java/io/ballerina/stdlib/http/api/client/endpoint/CreateSimpleHttpClient.java]: Resolved in [94eb323](https://github.com/ballerina-platform/module-ballerina-http/pull/673/commits/94eb323bb02ce7192e03872fa554f224df86b2bf)

File: ballerina-tests/tests/http_client_actions_test.bal [hunk 3]
```diff
@@ -635,3 +635,21 @@ public function testProxyClientError() {
         test:assertFail(msg = "Found unexpected output type");
     }
 }
+
+@test:Config {}
```
[the remark this answers is not in the document]
Comment (TharmiganK) [reply ballerina-tests/tests/http_client_actions_test.bal:639]: Resolved in [abc750e](https://github.com/ballerina-platform/module-ballerina-http/pull/673/commits/abc750e924677a9add27dc04cd1857fd2c3c3338)

**Review by chamil321 — APPROVED** [review]
